src/librssguard/services/standard/standardserviceroot.cpp

```diff
@@ -358,6 +358,24 @@ bool StandardServiceRoot::mergeImportExportModel(FeedsImportExportModel* model,
       }
       else if (source_item->kind() == RootItem::Kind::Feed) {
         auto* source_feed = qobject_cast<StandardFeed*>(source_item);
+        const auto items = target_root_node->childItems();
```

Comment:
Btw, this call is not "recursive", so if the goal is to "globally" eliminate import of duplicate feed URLs, then this code will not work. I will fix it.